crates/bevy_text/src/lib.rs

```diff
@@ -71,6 +71,10 @@ use bevy_render::{
 };
 use bevy_sprite::SpriteSystem;
 
+/// The raw data for the default font used by `bevy_text`
+#[cfg(feature = "default_font")]
+pub const DEFAULT_FONT: &[u8; 18848] = include_bytes!("FiraMono-subset.ttf");
```

Comment:
Shouldn't this be used internally somewhere?

Reply:
It's not

Reply:
It's done like this: https://github.com/bevyengine/bevy/blob/11ecc4d322c4c529ba40ff34eb8e35cc9e4da7fc/crates/bevy_text/src/lib.rs#L130

Reply:
In the future, would it be good to call:

```rust
const MY_FONT: &[u8] = include_bytes!("FireMono-subset.ttf");

load_internal_binary_asset!(
    app,
    Handle::default(),
    MY_FONT,
    |bytes: &[u8], _path: String| { Font::try_from_bytes(bytes.to_vec()).unwrap() }
); 
```

So that you could reference both the underlying `const` and the asset?